collect/central_collector.go

```diff
@@ -362,9 +362,12 @@ func (c *CentralCollector) AddSpan(span *types.Span) error {
 }
 
 func (c *CentralCollector) receive() error {
-	tickerDuration := c.Config.GetSendTickerValue()
```

Comment:
Do we need to update anything about SendTicker's config docs?

Reply:
I think we were overusing it conveniently for other things. We do need to update its docs for Refinery 3, but that's already planned work.